ras/ras.py

```diff
@@ -146,7 +146,9 @@ def test6_lsslot(self):
                       "==")
         self.run_cmd("lsslot")
         self.run_cmd("lsslot -c mem")
-        self.run_cmd("lsslot -ac pci")
+        slot = self.run_cmd_out("lsslot -ac pci")
+        if not slot:
+            self.log.info("There are no pci slots available")
```

Comment:
@shirishaganta I would have it is:

```
if self.run_cmd_out('lspci'):
    self.run_cmd_out("lsslot -ac pci")
```

Reply:
Yes @narasimhan-v  it is working

Reply:
Can the code be changed to this @shirishaganta  ?

Reply:
yes. Shall I change it?

Reply:
Yes please